src/lib/slider/slider.ts

```diff
@@ -295,6 +296,7 @@ export class MdSlider implements ControlValueAccessor {
   }
 
   @Output() change = new EventEmitter<MdSliderChange>();
+  @Output() input = new EventEmitter<MdSliderChange>();
```

Comment:
FYI: The JSDoc is intentionally missing here, because otherwise it might collide with #2321